tests/unit/src/test/scala/tests/debug/StackFrameDapSuite.scala

```diff
@@ -62,4 +62,29 @@ class StackFrameDapSuite
       )
     )
   )
+
+  assertStackFrame("overridden-toString")(
```

Comment:
We can use Bloop's newest snapshot version and there will be no need to separate this out.

Reply:
That's a good idea! I am trying that now.